examples/nlp/machine_translation/conf/aayn_finetune.yaml

```diff
@@ -0,0 +1,82 @@
+name: AttentionIsAllYouNeedFinetune
+do_training: True # set to False if only preprocessing data
+do_testing: False # set to True to run evaluation on test data after training
+model_path: test.nemo
```

Comment:
Can we make this a mandatory arg by making this ???

Reply:
Done!